### ISSUE TITLE
ngcart directive

### ISSUE THREAD

**Issue by kostalihas** [snapjay/ngCart#90]
I create a new directive and passe the same content of ngcartCart directive and just change the template url because I need another form html for my cart, I dont know why I get nothing displayed.

**Comment by g4b0-88** [comment]
Hi!
I also created a custom directive as follows:
`.directive('ngcartCart', [function(){
        return {
            restrict : 'E',
            controller : 'CartController',
            scope: {},
            templateUrl: function(element, attrs) {
                if ( typeof attrs.templateUrl == 'undefined' ) {
                    return 'assets/js/template_cart/ngCart/cart.html';
                } else {
                    return attrs.templateUrl;
                }
            },
            link:function(scope, element, attrs){

            }
        };
    }])`

And then just call it using `<ngcart-total></ngcart-total>`
Hope this helps!

**Comment by kostalihas** [comment]
yes it's a good idea ,another way I found it is to create a template.html and call it inside any directive using the attribute temlpate-url like bellow:
<ngcart-addtocart template-url="/template/ngCart/cartpdj.html"></ngcart-addtocart>
